library/sketches-library/src/test/java/uk/gov/gchq/gaffer/sketches/clearspring/cardinality/serialisation/HyperLogLogPlusSerialiserTest.java

```diff
@@ -48,27 +47,6 @@ protected Long getTestValue(final HyperLogLogPlus object) {
         return object.cardinality();
     }
 
-    @Test
-    public void testSerialiseAndDeserialiseWhenEmpty() {
```

Comment:
Is this removed because the new test file `HllSketchJsonDeserialiserTest.java` covers this now?

Reply:
Because it extends `ViaCalculatedValueSerialiserTest` it is superfluous